Jenkins_setup.md

```diff
@@ -0,0 +1,161 @@
+# Kata Containers Jenkins CI setup
+
+The main Continuous Integration (CI) system for Kata Containers is [Jenkins](https://jenkins.io/).
+This document gives an overview of how Kata Containers integrates with Jenkins, and
+provides pointers and information to configuration information and scripts that will
+help set up a Kata Containers Jenkins CI system.
+
+## Types of jobs
+
+Kata Containers CI operates two types of CI builds:
+
+- PR builds. Builds are scheduled upon submission or change to a Pull Request on any
+of the repositories
+
+- Master builds. A build is scheduled upon the merge of a Pull Request into the master
+branch of a repository.
+
+How these builds are configured and executed are subtly different:
+
+### PR builds
+
+PR builds are triggered from Jenkins using the [GitHub Pull Request Builder](https://plugins.jenkins.io/ghprb) (GHPRB)
+plugin.
+
+The GHPRB plugin is configured to be triggered via github hooks. This is the preferred method
+to trigger on changes (preferred to polling). The preferred method is to integrate the CIs
+using the [katacontainersbot](https://github.com/katacontainersbot) user. Please discuss
+your requirements with the Kata Containers team who can then discuss the appropriate changes.
```

Comment:
your requirements with the Kata Containers team who will then discuss the appropriate changes.